analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/SyntaxLightupGenerator.cs

```diff
@@ -1237,8 +1237,8 @@ private sealed class NodeData
                 nameof(AnonymousFunctionExpressionSyntax),
                 nameof(LambdaExpressionSyntax),
                 nameof(ParenthesizedLambdaExpressionSyntax),
-                nameof(SimpleLambdaExpressionSyntax)
-                );
+                nameof(SimpleLambdaExpressionSyntax),
+                nameof(ClassDeclarationSyntax));
```

Comment:
Should we include "TypeDeclaration" and "StructDeclaration" as well?

Reply:
I would not do it beforehand, once it's needed we might extend this list.
Why? From now onward we will create `TypeDeclarationSyntaxWrapper` that will show up together with `TypeDeclarationSyntax` in intellisense hints and available types. This might cause some confusion and why and when to use the wrapper IMO.